plugins/hls-tactics-plugin/src/Wingman/Judgements.hs

```diff
@@ -34,7 +34,7 @@ buildHypothesis
   where
     go (occName -> occ, t)
       | Just ty <- t
-      , isAlpha . head . occNameString $ occ = Just $ HyInfo occ UserPrv $ CType ty
+      , maybe False isAlpha . listToMaybe . occNameString $ occ = Just $ HyInfo occ UserPrv $ CType ty
```

Comment:
this is fine, but you might also like this version. Pattern guards are cool
```suggestion
      , (h:_) <- occNameString $ occ
      isAlpha h = Just $ HyInfo occ UserPrv $ CType ty
```

Reply:
Good point, I'm not used to using pattern guards so it didn't occur to me. Applied.